azure-pipelines-integration.yml

```diff
@@ -59,7 +59,7 @@ jobs:
         ArtifactName: '$(System.JobAttempt)-Logs $(_configuration) OOP64_$(_oop64bit) $(Build.BuildNumber)'
         publishLocation: Container
       continueOnError: true
-      condition: not(succeeded())
```

Comment:
How can the servicehub log be published? Or we need some other way to view it?

Reply:
Right, it gets attached as a build artifact